.ebextensions/pip.config

```diff
@@ -1,9 +1,9 @@
 container_commands:
   01_upgrade_pip:
-    command: "python -m pip install --upgrade pip"
+    command: "source /var/app/venv/*/bin/activate && python -m pip install --upgrade pip"
     leader_only: true
-  02_upgradde_setuptools:
-    command: "python -m pip install --upgrade setuptools"
+  02_upgrade_setuptools:
+    command: "source /var/app/venv/*/bin/activate && python -m pip install --upgrade setuptools"
```

Comment:
Is this still recommended?

Reply:
I got this from https://stackoverflow.com/questions/62457165/deploying-django-to-elastic-beanstalk-migrations-failed/63074781#63074781 and the answer claims that 

> I have verified with AWS Support that this is in fact the recommended way to do this. You must source the python environment, as with AL2 they use virtual environments in an effort to stay more consistent.

Reply:
OK sounds like it is indeed recommended.